lib/modules/sections.mjs

```diff
@@ -691,7 +691,39 @@ export async function validateIANAConsiderationsSection (doc, { mode = MODES.NOR
 
   switch (doc.type) {
     case 'txt': {
-      // TODO: Text type validation
+      const IANAConsiderationsSection = doc.data.content.ianaConsiderations
+
+      if (!IANAConsiderationsSection.length) {
```

Comment:
add there !IANAConsiderationsSection?.length, because if there is no iana consideration section in document- it throws error
![image](https://github.com/user-attachments/assets/5fe5789d-0050-42f4-bece-97c1495a85cb)